README.md

```diff
@@ -59,6 +59,8 @@ All parameters need to be provided as environment variables:
 | DD_PRODUCT_TYPE_NAME                | Mandatory          | Mandatory        | If a product type with this name does not exist, it will be created                               |
 | DD_PRODUCT_NAME                     | Mandatory          | Mandatory        | If a product with this name does not exist, it will be created                                    |
 | DD_ENGAGEMENT_NAME                  | Mandatory          | -                | If an engagement with this name does not exist for the given product, it will be created          |
+| DD_ENGAGEMENT_VERSION                 | Optional          | -                | If provided, the version is used as an additional filter to the name to find the matching engagement          |
+| DD_ENGAGEMENT_DEDUPLICATION                 | Optional          | -                | Default: false          |
```

Comment:
What did you need a deduplication flag?

Reply:
![image](https://github.com/user-attachments/assets/482aa1ba-31f2-42fb-a4b9-98ddf55531b0)

Different version of a "product" (redact-pipeline, redact-infer) will be tracked as different engaements. I don't want defect dojo to count identical finding across engagements as duplicates. Or otherway around each engagementt representing a differentt version should have it's independent findings.